### ISSUE TITLE
trim dependencies

### ISSUE THREAD

**Issue by sheridar** [rnabioco/djvdj#101]
* move as many as possible to Suggests and use conditionally

**Comment by jayhesselberth** [comment]
convert all pipes to `|>` and eliminate magrittr import

**Comment by jayhesselberth** [comment]
The number of dependencies is triggering a NOTE during R CMD CHECK:

```
* checking package dependencies ... NOTE
Imports includes 27 non-default packages.
Importing from so many packages makes the package vulnerable to any of
them becoming unavailable.  Move as many as possible to Suggests and
use conditionally.
```

**Comment by sheridar** [comment]
closed with #128
